### PR TITLE
Chris fixes for PR

### PR DESCRIPTION
**Pull request by ghop02**
Adds fixes for PR static value aggregation + adds a helper flag that skips all dataset filters

### CODE REVIEW

File: cli/data.py [hunk 1]
```diff
@@ -66,6 +66,37 @@ def main():
     pass
 
 
+def _run_dataset_filters(
```
Comment (TomGoBravo) [inline cli/data.py:69]: Good enough for now. Longer term I'd rather have a way to keep a copy of before and after the filters, than be able to see what changed at those points of the pipeline as input data and/or code changes.

Comment (ghop02) [reply cli/data.py:69]: 👍 i could imagine a world where this is 3 steps with separate outputs - combine, filter, aggregate

File: tests/libs/datasets/custom_aggregations_test.py [hunk 2]
```diff
@@ -78,3 +78,28 @@ def test_calculate_puerto_rico_bed_occupancy_rate():
         )
     )
     test_helpers.assert_dataset_like(actual, expected)
+
+
+def test_pr_aggregation():
+    dataset = combined_datasets.load_us_timeseries_dataset()
```
Comment (TomGoBravo) [inline tests/libs/datasets/custom_aggregations_test.py:84]: i'll likely exclude these from the fast tests but *much* better than nothing.

**Review by TomGoBravo — APPROVED** [review]
Thank you for adding these.

**Pull request merged**